docs/quickstart.rst

```diff
@@ -0,0 +1,16 @@
+ABC-Classroom Quickstart
+------------------------
+
+To set up a template directory to run abc-classroom in, run:
+
+    $ abc-quickstart
+
+This will create a directory for you that contains pre-made directories for templates and cloned files, as well as
+a sample ``config.yml`` file that can be modified to run the program for your classroom. ``abc-quickstart`` has
+two arguments that can be used to modify its functionality.
+
+1. ``--course_name course-repo-custom-name-here`` This argument will allow you to modify the name of the main
```

Comment:
ok i haven't tried this yet! lemme try it next!!

Reply:
HAHAHA you are cracking me up today with your github jokes today @nkorinek  it worked so i can resolve this!